recipes/vapor/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "vapor" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "6d7818c6556d24663ffa72352a887f3e236f10bff4559bd86e4081c7cccbe8af" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/connor-lab/vapor/archive/refs/tags/VAPOR_1.0.2.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed --no-cache-dir -vvv ."
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >= 3
+    - pip
+  run:
+    - python >= 3
+    - numpy >= 1.5.1
+
+test:
+  commands:
+    - vapor.py -h
+
+about:
+  home: "https://github.com/connor-lab/vapor"
+  license: GNU
+  license_family: GNU
```

Comment:
Try GPL here

Reply:
you can also add a `license_file: COPYING` which references the file in the github repo.

Reply:
ping